demisto_sdk/common/hook_validations/structure.py

```diff
@@ -45,7 +45,7 @@ class StructureValidator:
         'canvas-context-connections': JSON_ALL_CONNECTIONS_REGEXES,
         'classifier': JSON_ALL_CLASSIFIER_REGEXES,
         'layout': JSON_ALL_LAYOUT_REGEXES,
-        'incidentfield': JSON_ALL_INCIDENT_FIELD_REGEXES,
+        'incidentfield': JSON_ALL_INCIDENT_FIELD_REGEXES + JSON_ALL_INDICATOR_FIELDS_REGEXES
     }
```

Comment:
Why is `SCHEMA_TO_REGEX` redefined here if we have it in `demisto_sdk.common.constants`?

Reply:
I needed clarification from Hochman, and fixed it.